go-controller/pkg/ovn/ovn.go

```diff
@@ -670,6 +676,7 @@ func (oc *Controller) WatchCRD() {
 			crd := obj.(*apiextension.CustomResourceDefinition)
 			klog.Infof("Deleting CRD %s from cluster", crd.Name)
 			if crd.Name == egressfirewallCRD {
+				close(oc.egressFirewallDNS.stopChan)
```

Comment:
@JacobTanenbaum I would still have a Shutdown() method, or as we discussed, just let the egressDNS object continue to wake up every 30m and ignore shutting it down when the CRD goes away. That wakeup period is so long as to be no problem IMO.